Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -399,6 +559,171 @@ def get_scan_status_command():
     }
     return get_entry_for_object('Scan status for {}'.format(scan_id), 'TenableIO.Scan(val.Id === obj.Id)', scan_status)
 
+# TEST
+
+
+def pause_scan_command():
+    """Function for integration command pause_scan_command."""
+    scan_id = demisto.getArg('scanId')
+    scan_details = send_scan_request(scan_id)
+    scan_status = {
+        'Id': scan_id,
+        'Status': scan_details['info']['status']
+    }
+
+    if scan_status["Status"].lower() == "running":
+        send_scan_request(scan_id, "pause", "POST")
+        paused_scan = {
+            "Id": scan_id,
+            "Status": "Pausing"
+        }
+
+        return get_entry_for_object(
+            "The requested scan was paused successfully", "TenableIO.Scan", replace_keys(paused_scan), ["Id", "Status"])
+
+    else:
+        return "Command 'tenable-io-pause-scan' cannot be called while scan status is {}".format(scan_status["Status"])
+
+
+def pause_scans_command():
+    """Function for integration command pause_scans_command."""
+    scan_ids = str(demisto.getArg('scanIds')).split(",")
+
+    results = []
+
+    for scan_id in scan_ids:
+
+        scan_details = send_scan_request(scan_id)
+        scan_status = {
+            'Id': scan_id,
+            'Status': scan_details['info']['status']
+        }
+
+        if scan_status["Status"].lower() == "running":
+            send_scan_request(scan_id, "pause", "POST")
+            resumed_scan = {
+                "Id": scan_id,
+                "Status": "Pausing"
+            }
+            results.append(get_entry_for_object("The requested scan was paused successfully",
+                                                "TenableIO.Scan", replace_keys(resumed_scan), ["Id", "Status"]))
+
+        else:
+            results.append(
+                "Command 'tenable-io-pause-scans' cannot be "
+                "called while scan status is {} for scanID {}".format(scan_status["Status"], scan_id))
+
+    return results
+
+
+# TEST
```

Comment:
Please remove this comment

Reply:
Done
Update 02.23.2021 1034 HRS EST:
Removed comment from code